src/renderer/components/RuntimeAdapter.tsx

```diff
@@ -42,15 +45,17 @@ export default function RuntimeAdapter({
       async ({ accountId: notificationAccountId, msgId, chatId }) => {
         if (accountId !== notificationAccountId) {
           closeAllDialogs()
+          // selectAccount always loads the last used chat
+          // by setting saveLastChatId we force to load immediately
+          // the chat holding the notification message
```

Comment:
I don't understand this comment.

Reply:
ah is this meant as an optimisation? save it there first so it's faster?

Reply:
When a new account is selected - the first thing in MainScreen is useSelectLastChat which is not needed since we load (possibly) another chat (of the notification message)
So if we set LastChat ID before, the correct chat will be loaded there

It is a hack, but the alternative would be to set some jumpToMessage in ScreenContext that would be accessible in MainScreen before loading lastChat

If you have ideas how to clarify that in the comment please add them